### PR TITLE
Added conformance testing for binary primitive types.

### PR DESCRIPTION
**Pull request by haberman**
This is basic and more tests will be added over time.

### CODE REVIEW

**Comment by bazel-io** [comment]
Can one of the admins verify this patch?

**Comment by haberman** [comment]
Review to @xfxyjwf . This first CL is basic, but should be a good start. More tests to come.

**Comment by haberman** [comment]
All tests are passing @xfxyjwf .

File: conformance/conformance_test.cc [hunk 1]
```diff
@@ -675,6 +715,86 @@ bool ConformanceTestSuite::RunSuite(ConformanceTestRunner* runner,
     TestPrematureEOFForType(static_cast<FieldDescriptor::Type>(i));
   }
 
+  int64 kInt64Min = -9223372036854775808ULL;
```
Comment (xfxyjwf) [inline conformance/conformance_test.cc]: ULL => LL?

Comment (haberman) [inline conformance/conformance_test.cc]: You would think so. However the `ULL` is applied before the `-` (the `-` is not actually part of the constant). And `ABS(kInt64Min)` is too large to fit in an int64, so this throws compiler warnings. Some authors get around this by writing it as: `-9223372036854775807LL - 1`.

**Pull request merged**